index.html

```diff
@@ -416,6 +424,13 @@
       the <a>context object</a>'s <a>next sibling</a>.
     </dl>
   </ol>
+  
+  <p class=note>No special handling for <code><a>template</a></code> elements is included in the 
+  above "<code>afterbegin</code>" and "<code>beforeend</code>" cases. Unlike 
+  <a for="Element">innerHTML</a>, <a for="Element">insertAdjacentHTML</a> will insert the given 
+  <var>fragment</var> into a <code><a>template</a></code>'s <a>children</a> rather than into its
+  <a>template content</a>.</p>
```

Comment:
You are right, noob me forgot about interoperability and leaving my sense of logic at home.